iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/ModificationReports.java

```diff
@@ -352,7 +352,7 @@ static void removedVoltageLevelReport(Reporter reporter, String voltageLevelId)
                 .withKey("removeVoltageLevel")
                 .withDefaultMessage("Voltage level ${voltageLevelId}, its equipments and the branches it is connected to have been removed")
                 .withValue(VOLTAGE_LEVEL_ID, voltageLevelId)
-                .withSeverity(TypedValue.ERROR_SEVERITY)
+                .withSeverity(TypedValue.INFO_SEVERITY)
                 .build());
```

Comment:
The methods could be moved to the top of the file with the other "info" reports.

Reply:
Yes done